specs/test_formats/README.md

```diff
@@ -176,6 +176,18 @@ To prevent parsing of hundreds of different YAML files to test a specific test t
 ...                           <--- more test types
 ```
 
+## Common test-case properties
+
+Some test-case formats share some common key-value pair patterns, and these are documented here:
+
+```
+bls_setting: int     -- optional, can have 3 different values:
```

Comment:
`description` should also be in the "Common test-case properties" if it's required in every test format.

Reply:
Yes, but it's formatting of the test-format docs, not the format itself. So I'd rather not hotfix unnecessarily, and merge it with other test-format improvements later on (I hope to get & implement feedback from implementers on it as well).